charts/pega/charts/installer/values.yaml

```diff
@@ -69,6 +69,9 @@ upgrade:
   runRulesetCleanup: "false"
   # Rebuild Database Rules Indexes after Rules Load to improve Database Access Performance
   rebuildIndexes: "false"
+  # This flag has to be set for CMC use cases if hazelcast is running in client server mode. Default is false.Set it to true for HZ c/s mode.
```

Comment:
can we clean this up a bit?
Set this parameter to 'true' when your deployment uses Hazelcast in a client/server model. Default is 'false'. 
This setting is used to ensure the Pega Installer connects to the Hazelcast cluster running in a client/server model during application data updates.